ANSWERS.md

```diff
@@ -1,6 +1,14 @@
 ### Questions:
 1. What is the DOM?
+    the DOM is the document object model, which is an application
+    that logically shows HTML, and gives us an understandable way to 
+    manipulate.
 2. What is an event?
+    An event is an occurence on an element that interacts with
+    the user, like a click or mouseover.
 3. What is an event listener?
+    An event listenter is actively waits for the user to perform a specific action and then does something...like double clicking.
 4. Why would we convert a NodeList into an Array?
+    We convert nodelists to access array methods, like map.
```

Comment:
Answers!! Thanks for getting them done Rick 😄 

And that is exactly why we convert them into an array!